### DISCUSSION TITLE
BloatyNosyApp 0.85

### DISCUSSION THREAD

**Discussion by Belim** [builtbybel/Bloatynosy#66] [Announcements]
# Try the new [JunkCtrl app](https://github.com/builtbybel/JunkCtrl/releases)

- Optimized the integrated "bloatware" scanner. If you only appreciate this function of BloatyNosy, you can also use the standalone tool [JunkCtrl for W11](https://github.com/builtbybel/JunkCtrl) for it. It is more modular in terms of functionality and more aggressive in searching.
- Integrated a back button into the main form.
- Many minor optimizations.

**[!] INFO to all those still using the deprecated Microsoft Store version**. Please manually uninstall that version and switch to the open-source GitHub version. Development on the Store version has been discontinued, and the update mechanism no longer works. Some features are also no longer accessible.

I now also have an official statement from Microsoft explaining why the app was removed from the Store: _'Your submission does not have a valid code signing certificate.'_ This was definitely not correct under the old Store policy. They must have recently readjusted the policies for Win32 packages. However, I've had enough with the Store anyway. Unfortunately, every scam app manages to get in. The foolish ones are the developers who also invest their energy in an operating system to make it more appealing to a few more customers

<hr /><em>This discussion was created from the release <a href='https://github.com/builtbybel/BloatyNosy/releases/tag/0.85.0'>BloatyNosyApp 0.85</a>.</em>

**Comment by Byburhan2010** [comment]
I want to give Turkish language support

**Comment by calex85** [comment]
Hi, I just want to report an error on a fresh installation of Window 11.
![bloatynosy error](https://github.com/builtbybel/BloatyNosy/assets/113834941/17404415-57ec-4b41-8eae-fd52828686b4)
[Bloatynosy error.txt](https://github.com/builtbybel/BloatyNosy/files/12231282/Bloatynosy.error.txt)


Thanks for all your work.

**Reply by Protons1** [reply]
Run as administrator

**Comment by wlsontan** [comment]
After debloating, my snippingtool wont popup after pressing PrtSc Key, it should have opened and showed an screenshot popup, Help please

**Comment by wepajoli** [comment]
Thanks for this awesome tool! 
I tried to install with the Hexnode UEM with the msi installer, but I ran into an issue. 
Can I have some help? Thanks

![Capture d'écran 2023-08-23 052922](https://github.com/builtbybel/BloatyNosy/assets/64052069/39d9c092-2a0e-49b2-a560-d7b482752901)
